ee/modules/030-cloud-provider-openstack/images/cloud-controller-manager/werf.inc.yaml

```diff
@@ -53,5 +45,12 @@ shell:
   - cd /src
   - test -d /patches/{{ $version }} && for patchfile in /patches/{{ $version }}/*.patch ; do patch -p1 < ${patchfile}; done
   - make VERSION={{ $value.ccm.openstack }} GIT_VERSION={{ $value.ccm.openstack }} openstack-cloud-controller-manager
+  {{- if semverCompare ">=1.27" $version }}
+  - chown 64535:64535 /src/openstack-cloud-controller-manager
+  - chmod 0755 /src/openstack-cloud-controller-manager
+  {{- else }}
+  - chown 64535:64535 /src/openstack-cloud-controller-manager-amd64
+  - chmod 0755 /src/openstack-cloud-controller-manager-amd64
+  {{- end }}
```

Comment:
strange construction, simply move openstack-cloud-controller-manager-amd64 to openstack-cloud-controller-manager